client/components/sites-dropdown/style.scss

```diff
@@ -34,7 +34,7 @@
 
 	.has-multiple-sites &:hover {
 		border-color: $gray-lighten-10;
-		border-bottom-width: 2px;
+		box-shadow: 0 1px 0 $gray-lighten-10;
```

Comment:
We should be able to omit the third value (`0` [blur-radius](https://developer.mozilla.org/en-US/docs/Web/CSS/box-shadow#%3Cblur-radius%3E)) as it defaults to `0`:

```
box-shadow: 0 1px $gray-lighten-10;
```

Reply:
Indeed 👍 Omitted.